### ISSUE TITLE
Add classification to Monitor properties

### ISSUE THREAD

**Issue by flavio-cadete-cko** [DataDog/terraform-provider-datadog#1039]
Missing new Datadog Monitor property 'classification'
values e.g. 'log', 'metric', ...

**Comment by therve** [comment]
Hello,

Can you describe what you mean? I don't think classification is a parameter to any of our monitor APIs. Thanks.

**Comment by therve** [comment]
Feel free to reopen with more information.

**Comment by flavio-cadete-cko** [comment]
![image](https://user-images.githubusercontent.com/51323090/120320630-4dde3780-c2da-11eb-8410-b4a62b6aebb0.png)
![image](https://user-images.githubusercontent.com/51323090/120320548-37d07700-c2da-11eb-97fc-5955bddb49e8.png)
![image](https://user-images.githubusercontent.com/51323090/120320549-38690d80-c2da-11eb-85d8-9eefc460682b.png)

**Comment by therve** [comment]
Ah this is in the response, so I think it's read-only? I'll reach out to the monitor team.

**Comment by flavio-cadete-cko** [comment]
Yes, it's confusing the json property `type` as what is the actual Monitor type the classification.

**Comment by timogoosen** [comment]
I second this request
